.github/workflows/typos.toml

```diff
@@ -0,0 +1,19 @@
+[files]
+extend-exclude = [
+    "*.svg",
+    "*.png",
+    "*.json",
+    ]
+
+[default.extend-words]
+# Ignore IDAES
+IDAES = "IDAES"
+idaes = "idaes"
```

Comment:
I've just checked to be sure, and it seems corrections are not case-sensitive, so we don't need different capitalizations here.